agent_build_refactored/managed_packages/steps/build_python.sh

```diff
@@ -71,92 +69,53 @@ INCLUDE_DIR="include"
 	--enable-shared \
 	--with-openssl="/usr/local" \
 	--with-readline=edit \
-	--prefix="${PYTHON_INSTALL_PREFIX}" \
-	--enable-optimizations \
-	--with-lto \
-	--with-ensurepip=install \
-	--bindir="${PYTHON_INSTALL_PREFIX}/${BIN_DIR}/${SUBDIR_NAME}" \
-	--datarootdir="${PYTHON_INSTALL_PREFIX}/${DATA_ROOT_DIR}/${SUBDIR_NAME}" \
-	--includedir="${PYTHON_INSTALL_PREFIX}/${INCLUDE_DIR}/${SUBDIR_NAME}" \
-	--with-platlibdir="${LIB_DIR}/${SUBDIR_NAME}"
+	--prefix="${PACKAGE_INSTALL_PREFIX}" \
+	--exec-prefix="${PACKAGE_INSTALL_EXEC_PREFIX}" \
+	--with-ensurepip=upgrade \
+	--with-suffix="-orig"
+
+#		--enable-optimizations \
```

Comment:
Why were --enable-optimizations & --with-lto commented out?

Reply:
Correct, just to save time during the active development. Added reminder TODO to uncomment that for stable release.